### PR TITLE
VZ-1181: Improve OLCNE install documentation

### PR DESCRIPTION
**Pull request by kminder**
Adds context information to the OLCNE install about using OCI as the infrastructure for testing.

### CODE REVIEW

File: install/install-olcne.md [hunk 1]
```diff
@@ -36,200 +147,221 @@ A Verrazzano Oracle Linux Cloud Native Environment deployment requires:
...
         storageclass.kubernetes.io/is-default-class: "true"
     provisioner: kubernetes.io/no-provisioner
     volumeBindingMode: WaitForFirstConsumer
   EOF
   ```
-* Create a `PersistentVolume`
-  ```yaml
-  cat << EOF | kubectl apply -f -
+* Create the required number of `PersistentVolume` resources.
+  The Verrazzano system requires 5 persistent volumes for itself.
```
Comment (mark-au) [inline install/install-olcne.md:196]: Other sections of the document spell out numbers less than ten, I'd suggest 5, 4, 9 in this paragraph be done the same way

File: install/install-olcne.md [hunk 2]
```diff
@@ -1,32 +1,143 @@
...
-The following software must be installed on your deployment system:
-* curl
-* helm
-* jq
-* kubectl
-* openssl
+## Prerequisites
+* Verrazzano requires Oracle Linux Cloud Native Environment version 1.1 or later.
+* Verrazzano installation requires the following software be installed on the system used to install Verrazzano.
+  The OLCNE operator node is typically used to install Verrazzano.
```
Comment (mark-au) [inline install/install-olcne.md:11]: I'm told we're not allowed to abbreviate Oracle product names in documentation. I'd personally keep using OLCNE in your tables as that'd decrease readability. I'd reduce the number of times you use the term and spell it in its entirety when you need it

File: install/install-olcne.md [hunk 3]
```diff
@@ -36,200 +147,221 @@ A Verrazzano Oracle Linux Cloud Native Environment deployment requires:
...
-##### Configuration
-* Target Host: Hostnames of Kubernetes worker nodes
-* Target Port: See table
+  
+#### Load Balancers
+Verrazzano on Oracle Linux Cloud Native Environment can use external load balancer services.
+These will not automatically be provided by Verrazzano or Kubernetes.
+If desired two load balancers may be deployed outside of subnet used for the Kubernetes cluster.
+One load balancer is for management traffic and the other for application traffic.
+If load balancers are not created the management and application user interfaces will need to be accessed by IP address.
```
Comment (mark-au) [inline install/install-olcne.md:229]: I thought load balancers were now mandatory as the installation procedure makes calls to Rancher

**Review by mark-au — CHANGES_REQUESTED** [review]

File: install/install-olcne.md [hunk 4]
```diff
@@ -1,32 +1,143 @@
...
+      |No       |10.0.0.0/24|TCP     |All         |31390            |           |HTTPS Load Balancer  |
+      |No       |10.0.1.0/24|UDP     |All         |111              |           |NFS                  |
+      |No       |10.0.1.0/24|TCP     |All         |111              |           |NFS                  |
+      |No       |10.0.1.0/24|UDP     |All         |2048             |           |NFS                  |
+      |No       |10.0.1.0/24|TCP     |All         |2048-2050        |           |NFS                  |
+      |No       |10.0.1.0/24|TCP     |All         |2379-2380        |           |Kubernetes etcd      |
+      |No       |10.0.1.0/24|TCP     |All         |6443             |           |Kubernetes API Server|
+      |No       |10.0.1.0/24|TCP     |All         |6446             |           |MySQL                |
+      |No       |10.0.1.0/24|TCP     |All         |8090-8091        |           |OLCNE Platform Agent |
+      |No       |10.0.1.0/24|TCP     |All         |8472             |           |Flannel              |
```
[the remark this answers is not in the document]
Comment (kminder) [reply install/install-olcne.md:84]: Will fix.

File: install/install-olcne.md (hunk 1 above)
Comment (kminder) [reply install/install-olcne.md:196]: Will fix.

**Comment by kminder** [comment]
@mark-au < Made suggested changes.  Please re-review.

**Review by mark-au — APPROVED** [review]

**Pull request merged**